source/git/source_test.go

```diff
@@ -578,7 +586,7 @@ func setupGitRepo(t *testing.T) gitRepoFixture {
 		"echo sbb > foo13",
 		"git add foo13",
 		"git commit -m third",
-		"git tag lightweight-tag",
+		"git tag --no-sign lightweight-tag",
```

Comment:
Is it to skip GPG sign to suppress warning?

Reply:
Oops, this was from https://github.com/moby/buildkit/pull/3960/files#diff-a7d654e4edba8b86b4853225da6c0e915b401becf96b20c380ac7105c3850610R585 - I don't think this needs to change.

Reply:
Yeah, I added this because I was running the tests directly on my machine which has various things integrated with `git` for commit signing and such. Feel free to remove it if you only care about running the tests in a container.

Reply:
Aha, this is a good call IMO, I'll split this to a separate commit :tada: